### PR TITLE
Add integration tests for settle and claim

### PR DESCRIPTION
**Pull request by ajsantander**

### CODE REVIEW

File: test/integration/utils/settings.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+	let { SystemSettings } = ctx.contracts;
+	SystemSettings = SystemSettings.connect(ctx.users.owner);
+
+	const tx = await SystemSettings[
+		`set${settingName.charAt(0).toUpperCase()}${settingName.slice(1)}`
+	](newValue);
+	await tx.wait();
+}
+
+async function forceSetSystemSetting({ ctx, settingName, newValue }) {
```
Comment (ajsantander) [inline test/integration/utils/settings.js:19]: hackr

File: test/integration/utils/exchanging.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+	await tx.wait();
+}
+
+function ignoreFeePeriodDuration({ ctx }) {
+	before('record and reduce feePeriodDuration', async () => {
+		const { SystemSettings } = ctx.contracts;
+
+		ctx.feePeriodDuration = await SystemSettings.feePeriodDuration();
+
+		await forceSetSystemSetting({ ctx, settingName: 'feePeriodDuration', newValue: 5 });
```
Comment (justinjmoses) [inline test/integration/utils/exchanging.js:44]: worth explaining here with a comment why this is necessary

**Review by justinjmoses — APPROVED** [review]
Just one comment

File: test/integration/behaviors/stake.behavior.js [hunk 3]
```diff
@@ -37,6 +39,49 @@ function itCanMintAndBurn({ ctx }) {
...
+
+						it('shows a slight increase in the users sUSD balance', async () => {
+							assert.bnGt(await SynthsUSD.balanceOf(user.address), balancesUSD);
+						});
+					});
+				});
+			});
 		});
 
 		describe('when the user burns sUSD', () => {
```
Comment (leomassazza) [inline test/integration/behaviors/stake.behavior.js:87]: worths splitting issue and burn into two different behaviours of stake

**Pull request merged**